### PR TITLE
Feature database

### PR DESCRIPTION
**Pull request by JoeyKraemer**
check databasestructure.sql

### CODE REVIEW

File: DatabaseStructure.sql [hunk 1]
```diff
@@ -23,43 +23,44 @@ CREATE TABLE `Roles` (
...
    PRIMARY KEY(user_id),
-   FOREIGN KEY(type_of_staff) REFERENCES TypesOfStaff(type_of_staff_id) ON UPDATE CASCADE,
-   FOREIGN KEY(user_role) REFERENCES Roles(role_id) ON UPDATE CASCADE
+   FOREIGN KEY(type_of_staff) REFERENCES TypesOfStaff(type_of_staff_id) ON UPDATE CASCADE ON DELETE NO ACTION,
+   FOREIGN KEY(user_role) REFERENCES Roles(role_id) ON UPDATE CASCADE ON DELETE NO ACTION
 );
 
 -- Table Event_Details
-DROP TABLE IF EXISTS `Event_Details`;
 CREATE TABLE `Event_Details` (
```
Comment (YohanLoz) [inline DatabaseStructure.sql:57]: Needs a way to check whether a user is currently checked or has checked out already in (check out/check in variable in database).

**Review by YohanLoz — CHANGES_REQUESTED** [review]

File: DatabaseStructure.sql (hunk 1 above)
Comment (JoeyKraemer) [reply DatabaseStructure.sql:57]: Check in will be stated by creating the entry into the "event_details" table and checkout by deleting that entry

**Review by YohanLoz — CHANGES_REQUESTED** [review]
Event and User are singular while Types- and Roles are plural, i'm not sure if you have a reasoning but it seems inconsistent.
the inactive variable seems unintuative since it creates unnecessary negatives and sometimes double negatives ( inactive = false is harder to interperate than active = true ), using true as the default value should not be a concern. 
i'm also just going to point out that you made it so location street and city ect can not be null (and thus should always be filled in), but the client said that reporters might have to add events on the spot, and thus it would be useful to have a placeholder value like null to fill in later. this can of course be argued. same goes for event_max_participants, because sometimes it would be unreasonable to know, it's better as an optional variable.
i'm not sure if you can put a character limit on a TEXT type, but we don't want people to just copy paste the script of a certain movie about bees in there. so it would either have to be adressed in the database, in the entry code, or through the manager by fear of getting fired

please change the first plural/singular names and "inactive" and please send a message about the required locations and character limit

**Review by YohanLoz — APPROVED** [review]
All corrections have been accounted for.

**Pull request merged**